Makefile

```diff
@@ -94,7 +95,9 @@ K_JAR := $(K_SUBMODULE)/k-distribution/target/release/k/lib/java/kernel-1.0-SNAP
 deps: repo-deps
 repo-deps: k-deps plugin-deps
 k-deps: $(K_JAR)
-plugin-deps: $(PLUGIN_SUBMODULE)/client-c/main.cpp
+plugin-deps: $(PLUGIN_SUBMODULE)/client-c/main.cpp $(wildcard $(PLUGIN_SUBMODULE)/plugin/*)
```

Comment:
Why the added dependent (the `$(wildcard ...)` one)?

Reply:
The idea is that building the semantics depends on the K source files inside the blockchain plugin, i.e. everything under the `plugin/` subdir.